roles/openshift_manageiq/tasks/main.yaml

```diff
@@ -59,6 +59,9 @@
   register: oshawkular_create_cluster_role
   failed_when: "'already exists' not in oshawkular_create_cluster_role.stderr and oshawkular_create_cluster_role.rc != 0"
   changed_when: oshawkular_create_cluster_role.rc == 0
+  # AUDIT:changed_when_note: Checking the return code is insufficient
+  # here. We really need to verify the if the role even exists before
+  # we run this task.
```

Comment:
@tbielawa Yes this task will *always* run in the same way that a list task would *always* run. If the command/task already possesses some level of idempotency, why not take advantage of it?